### PR TITLE
Some basic cleanup and integrations from master

### PR DESCRIPTION
**Pull request by HoldYourWaffle**
I cleanup some of the code up in advance for something else i'm gonna change. Just thought I'll send the pr already :)

(Oh btw I also added the fr-language file from @Mazdallier and the readme from the master branch)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,266 @@
-/bin
-/build
-/.classpath
-/.project
-/.gradle
-/.settings
-/eclipse
+#################
```
Comment (olee) [inline .gitignore]: Just no.
What the heck did you do to the gitignore?
Revert it please.

File: src/main/java/de/gakai/flighttable/FlightTableMod.java [hunk 2]
```diff
@@ -91,19 +82,19 @@ public void init(FMLInitializationEvent event)
         GameRegistry.registerItem(creativeFeather, "creative_feather");
         GameRegistry.registerBlock(flightTable, "flight_table");
 
-        GameRegistry.addShapedRecipe(new ItemStack(flightTable), //
```
Comment (olee) [inline src/main/java/de/gakai/flighttable/FlightTableMod.java:94]: The comments were present to prevent line-merging from auto-formatting :wink:

**Comment by olee** [comment]
There are some unnecessary changes in there (especially the gitignore).
Also please rebase your changes on the current master first.

**Comment by HoldYourWaffle** [comment]
I think the gitignore messed up (unintentionally) when I'd setup my workspace. Reverted it.
The commenting-thingy I also reversed, didn't know that messed up things...

About the master thing; the master branch is currently on 1.7.10
(More mods have the same sitiuation, master is on 1.7.10 and the 1.8 has it's seperate branches, is there something special about that? Am I missing something?)

**Comment by olee** [comment]
The master is on 1.7.10 because it is still more widely used on modded servers.
Also it's easier to program on 1.7.10 and then just merge the changes into 1.8.

Also don't forget to rebase your changes.

**Comment by HoldYourWaffle** [comment]
Ok I get it now. It's quite logic actually if you think about it...

As I'm not that experienced with github and stuff, what would be the best way to rebase my changes? Delete my repo and fork again (and a new pr), or reverting all my commits?

**Comment by HoldYourWaffle** [comment]
I just deleted my fork and I'm gonna do it correctly this time ;)

**Pull request closed without merging**